packages/relay-compiler/bin/RelayCompilerBin.js

```diff
@@ -234,6 +221,10 @@ const argv = yargs
       describe: 'More verbose logging',
       type: 'boolean',
     },
+    'noWatchman': {
```

Comment:
Let's avoid double-negative options. How about just a `watchman` flag which defaults to true, so `--watchman=false` to disable?